overlayer/setup.py

```diff
@@ -7,7 +7,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-requirements = ['boto3', 'botocore', 'amazon-textract-caller>=0.0.11', 'Pillow>=9.2.*', 'PyPDF2>=2.5.*']
+requirements = ['boto3', 'botocore', 'amazon-textract-caller>=0.0.11', 'Pillow>=9.2.*', 'pypdf>=2.5.*']
```

Comment:
Why change PyPDF2?

Reply:
since the version will be depreciated https://pypi.org/project/PyPDF2/

NOTE: The PyPDF2 project is going back to its roots. PyPDF2==3.0.X will be the last version of PyPDF2. Development will continue with [pypdf==3.1.0](https://pypi.org/project/pyPdf/).